src/main/java/frc/robot/subsystems/Vision/VisionIO.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.vision;
+
+import edu.wpi.first.math.geometry.Pose3d;
+import java.util.ArrayList;
+import java.util.List;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+/** Add your docs here. */
+public interface VisionIO {
+
+  public static class VisionIOInputs {
+    public double timestamp = 0.0;
+    public double latency = 0.0;
```

Comment:
are these redundant?

Reply:
I don't think so, timestamp is like when in the match it happened and latency should just be latency